selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/git/GitRevertCommit.java

```diff
@@ -67,6 +67,13 @@ public void clickRevertButton() {
         .click();
   }
 
+  public void clickCancelButton() {
+    new WebDriverWait(seleniumWebDriver, ELEMENT_TIMEOUT_SEC)
```

Comment:
It's better to use helper class [WebDriverWaitFactory](instances: https://github.com/eclipse/che/blob/6.2.0/selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/WebDriverWaitFactory.java ) which aimed to avoid creation of enormous WebDriverWait instances.